global/definitions.php

```diff
@@ -48,7 +48,7 @@
 
 //Increment these versions whenever you update any js or css files for cachebusting
 define("JSVERSION",1.2);
-define("CSSVERSION",1.23);
+define("CSSVERSION",1.24);
```

Comment:
I think you need to update the version to 1.57 if you're pushing that same SQL version. We're on 1.56 right now.